src/components/UI/Alert/DialogBox.vue

```diff
@@ -117,6 +117,8 @@ export default {
       // is either of the confirm or cancel buttons visible
       // the only way to hide each of those two buttons is by
       // adding the `hidden` class to each of those button classes
+      if (this.confirmButtonClass == undefined || this.cancelButtonClass == undefined)
```

Comment:
why was this added?

Reply:
If for some reason `confirmButtonClass` or `cancelButtonClass` was undefined, then the code below, `this.confirmButtonClass.indexOf("hidden")` gave an error like `undefined has no method indexOf`